cli/command/service/logs.go

```diff
@@ -112,7 +112,7 @@ func runLogs(dockerCli *command.DockerCli, opts *logsOptions) error {
 			return err
```

Comment:
`responseBody, err := cli.TaskLogs(ctx, opts.target, options)`
Here, is `:=` OK? It cause `responseBody declared and not used`, it is better like
responseBody, err = cli.TaskLogs(ctx, opts.target, options)